scripts/Find_tracks_in_raw_data_from_start.m

```diff
@@ -0,0 +1,195 @@
+close all;
```

Comment:
If different from Find_tracks_in_raw_data_ARS.m, then comment in why, otherwise merge.